pages/common/elm.md

```diff
@@ -0,0 +1,15 @@
+# elm
+
+> Run and manage programs in Elm programming language.
+
+- Run interactive elm shell (repl):
+
+`elm --repl`
+
+- Initialize empty elm project:
```

Comment:
Perhaps `Initialize an empty`... would read better?

Reply:
Thanks, going to fix it